stint/io.nim

```diff
@@ -16,6 +16,10 @@ import
 
 from stew/byteutils import toHex
 
+# Helpers
+# --------------------------------------------------------
+{.push raises: [], inline, gcsafe.}
```

Comment:
I think ``inline`` should be selectively added here.

Reply:
Fixed